brave/brave-core
# (Wallet) Display network logo drawables in network selection screen during Wallet creation

## Resolves https://github.com/brave/brave-browser/issues/37764

<img src="https://github.com/brave/brave-core/assets/3308503/ceeb2fb8-fc1d-4012-9816-0ff3273d8031" width="300" />

Display network logo drawables instead of PNG files. 
Network logo PNG files are part of the Wallet data, thus are downloaded _after_ the Wallet is created and during onboarding process are not available.

Vector drawables of network logos have been exported from Nala repository except for these (they will be added soon to Nala):
* Neon
* Arb
* Op



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Start with a fresh install
* Create a new Wallet
* Navigate to network selection screen
* Observe network logos are displayed correctly
